server/utils/config.js

```diff
@@ -11,11 +11,17 @@ function createDatabaseUrl() {
 	if (process.env.DATABASE_URL) {
 		return process.env.DATABASE_URL;
 	}
-	const host = process.env.DB_HOST ?? "localhost";
-	const name = process.env.DB_NAME ?? "cyf";
-	const password = process.env.DB_PASS ?? process.env.DB_PASSWORD ?? "";
+	const host =
```

Comment:
would prefer you not add the passwords in the database directly in your commits, rather have it in your env file, you can reach out to @ChrisKoetzee since he had the same issue or me